### ISSUE TITLE
Move React and React Native in package.json from dependencies to peerDependencies

### ISSUE THREAD

**Issue by ericnograles** [MiLeung/done-bar#7]
Hi there!

We happen to use this library in one of our projects (nice job btw 👍 ), but I did notice something that doesn't jive with the latest versions of React Native.

I see your `package.json` locks down `react` and `react-native` to specific versions of both.  Unfortunately, when trying to run this with the latest version of RN, it will cause issues.  You'll get a weird RSOD that seems to indicate that `ActivityIndicator` can't be resolved, assuming you import it in from `react-native`.

The reason for this is because this library pulls in RN 0.40 _on top of my actual version of React Native_.  This causes some collisions and the dependencies get jacked up.  Please see this RN issue for reference: https://github.com/facebook/react-native/issues/14302

In that issue, the `react-native-calendar-picker` was causing grief for that particular developer because it locked down the version of RN.

I'd suggest adding a little flexibility to your package.json.  Stealing from AirBnB's `react-native-maps` library, you might want to refactor your package.json by moving your `react-native` and `react` from dependencies to peerDependencies like so:

```
"peerDependencies": {
    "react": ">=15.4.0",
    "react-native": ">=0.40"
  }
```

I'll submit a PR shortly for this, but thought I'd let you know.

Thank you!

**Comment by mileung** [comment]
Thanks!  Merged your pull request just now!

**Comment by smilingkylan** [comment]
@MiLeung It looks like this change was not merged into whatever version `npm install done-button` installs. Or am I doing something wrong?

**Comment by mileung** [comment]
@kylanhurt Probably my fault, but no one has mentioned this issue until now. Anyways, I just published version 1.0.4, so it should definitely work now.
